istioctl/pkg/install/verify_test.go

```diff
@@ -55,10 +55,10 @@ var (
 		},
 		Status: v1beta1.DeploymentStatus{
 			Conditions: []v1beta1.DeploymentCondition{
-				v1beta1.DeploymentCondition{
+				{
```

Comment:
What is the rationale for removing types that the compiler can detect implicitly?  Won't the code be harder to search with this change?

Reply:
It fits with the practice of not having dummy variables. In Go, you do:

  _ = MyFunc()

rather than

  dummy = MyFunc()

The _ explicitly tells the reader "this value is being ignored".